packages/client/src/views/PrintCertificate/PDFUtils.ts

```diff
@@ -98,7 +98,20 @@ export function executeHandlebarsTemplate(
     },
     cache
   )
-  registerHandlebarHelpers()
+
+  const customHelpers = getHandlebarHelpers()
+
+  for (const helperName of Object.keys(customHelpers)) {
+    /*
+     * Note for anyone adding new context variables to handlebar helpers:
+     * Everything you expose to country config's here will become API surface area,
+     * This means that countries will become dependant on it and it will be hard to remove or rename later on.
+     * If you need to expose the full record, please consider only exposing the specific values you know are needed.
+     * Otherwise what happens is that we lose the ability to refactor and remove things later on.
+     */
```

Comment:
Should this warning rather be in Farajaland, or are the context variables added to this file?

Reply:
@naftis they are exposed from here, yes 👍